website/docs/installation/configuration.mdx

```diff
@@ -144,19 +144,19 @@ These settings affect where media files are stored. Those files include applicat
 -   `AUTHENTIK_STORAGE__MEDIA__S3__SECURITY_TOKEN`: Security token to authenticate to S3. May be omitted. Supports hot-reloading.
 -   `AUTHENTIK_STORAGE__MEDIA__S3__BUCKET_NAME`: Name of the bucket to use to store files.
 -   `AUTHENTIK_STORAGE__MEDIA__S3__CUSTOM_DOMAIN`: Domain to use to create URLs for users. Mainly useful for non-AWS providers. May include a port. Must include the bucket. Example: `s3.company:8080/authentik-media`.
--   `AUTHENTIK_STORAGE__MEDIA__S3__SECURE_URLS`: Whether URLS created for users use `http` or `https`. Defaults to `true`.
+-   `AUTHENTIK_STORAGE__MEDIA__S3__SECURE_URLS`: Whether URLS created for users use `https` rather than `http`. Defaults to `true`.
```

Comment:
Well, it's `http` rather than `https` as the default is True.

Reply:
At least I was correct in thinking the phrasing was potentially confusing there 😆

Reply:
...I think I misunderstood your comment, unless `SECURE_URLS=true` actually means 'use insecure http'. To make it even clearer, how about:

> Whether URLS created for users use `https` rather than `http`. Defaults to `true` to use HTTPS.

Or if the alternative is clear, just simplify as:

> Whether URLS created for users use `https`. Defaults to `true`.

Reply:
SECURE_URLS=true -> https

SECURE_URLS=false -> http

Whatever you think is less ambiguous is fine by me.

Reply:
Proposal:
-   `AUTHENTIK_STORAGE__MEDIA__S3__SECURE_URLS`: This option determines if URLs created use `https` (set to `true` by default) or `http`.